projects/igniteui-angular/src/lib/services/overlay/position/IPositionStrategy.ts

```diff
@@ -0,0 +1,9 @@
+import { PositionSettings } from './../utilities';
+
+export interface IPositionStrategy {
+    // _wrapperClass: string;
+    _settings: PositionSettings;
```

Comment:
This shouldn't exist on the interface. Assuming you do the target setting merge + util method each strategy can call in, the `_settings` becomes internal for each strategy as nothing else would need to use those, right?

Reply:
we need this at least to reach to animation in show and hide methods of overlay.
Also one can easily setup position settings via settings property, and not by providing settings via the constructor.